### DISCUSSION TITLE
I don't know how to delete a discussion so ignore this

### DISCUSSION THREAD

**Discussion by qfmp1012** [niels747/2D-Weather-Sandbox#84] [Q&A] [answered]
Is there a way to simulate an extratropical cyclone Moving through the map? I am simulating a town that is south of Tasmania and it would be nice to simulate extratropical cyclones.

**Comment by coolkid0281** [comment, marked as answer]
That would be really cool, But I don't know if there are :(

**Comment by Westerstaad** [comment]
for the same reason it is not possible to simulate a hurricane, tornado, or any other rotating storm, it is not possible to have a extratropical cyclone in a 2d space because rotating storms need 3 dimension to be able to rotate vertically
